packages/keycloak-user-management/package.json

```diff
@@ -16,6 +17,7 @@
     "fast_array_intersect": "^1.1.0",
     "formik": "^2.1.5",
     "formik-antd": "^2.0.1",
+    "history": "^5.0.0",
```

Comment:
@kahummer how is this being used?

Reply:
@p-netm It's being used on various test cases
https://github.com/OpenSRP/web/blob/135-user-management-ui-update/packages/keycloak-user-management/src/components/Credentials/tests/index.test.tsx#L3 among others.
Tried using useHistory hook but seems not to be supported outside the functional components.

Reply:
@p-netm it was added prior to using usehistory hook. I've removed the same